src/pages/projects/form/ProjectFormDialog.tsx

```diff
@@ -130,10 +130,13 @@ export const ProjectFormDialog = ({
                 coordinatingInstitution: {
                   id: true,
                 },
-                contributors: values.contributors.map(() => ({
-                  type: true,
+                contributors: values.contributors.map((contributor) => ({
                   identity: { id: true },
-                  affiliation: { id: true },
+                  roles: contributor.roles.map(() => ({
+                    affiliation: {
+                      id: true,
+                    },
+                  })),
```

Comment:
New structure for the touched-object to fit the new API backend. This is used when detecting if a user has visited a field, and thus if an error message should be displayed.